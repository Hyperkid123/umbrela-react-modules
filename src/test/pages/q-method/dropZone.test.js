import React from 'react'
import DropZone from '../../../pages/q-method/dropZone';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';

describe('Q-method dropZone component', () => {
  it('should render correclty' , () => {
      const tree = shallow(
        <DropZone/>
      ).dive();
      expect(toJson(tree)).toMatchSnapshot();
  });
});
