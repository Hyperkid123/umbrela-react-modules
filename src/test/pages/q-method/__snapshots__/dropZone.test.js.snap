// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Q-method dropZone component should render correclty 1`] = `
<div
  className="sc-bwzfXH bAwxlR"
>
  <Connect(ImageDropZone) />
</div>
`;
